projects/plugins/jetpack/_inc/client/state/site/reducer.js

```diff
@@ -412,3 +423,26 @@ export function getConnectedPlugins( state ) {
 	const plugins = get( state.jetpack.siteData, [ 'data', 'site', 'connectedPlugins' ], [] );
 	return plugins.filter( plugin => 'jetpack' !== plugin.slug );
 }
+
+/**
+ * Returns true if the site has a subscription to a Backup product or to a plan that includes Backup.
+ *
+ * @param   {object} state - Global state tree
+ * @returns {boolean} True if the site does have Backup
+ */
+export function siteHasBackup( state ) {
+	const sitePlan = getSitePlan( state );
+	const siteProducts = getSiteProducts( state );
+
+	let hasBackup = false;
+
+	if ( sitePlan && sitePlan.product_slug ) {
+		hasBackup = hasBackup || isJetpackPlanWithBackup( sitePlan.product_slug );
+	}
+
+	if ( Array.isArray( siteProducts ) ) {
+		hasBackup = hasBackup || siteProducts.some( p => isJetpackProduct( p.product_slug ) );
+	}
```

Comment:
To avoid calling `Array.isArray()` every time, how about this?

```suggestion
	if ( ! hasBackup && Array.isArray( siteProducts ) ) {
		hasBackup = siteProducts.some( p => isJetpackProduct( p.product_slug ) );
	}
```